www/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "paragon-pattern-library-documentation",
+  "description": "Paragon Pattern Library Documentation",
+  "version": "1.0.0",
+  "dependencies": {
+    "@mdx-js/mdx": "^1.0.10",
+    "@mdx-js/react": "^1.0.6",
+    "gatsby": "^2.3.24",
+    "gatsby-mdx": "^0.6.2",
+    "gatsby-plugin-manifest": "^2.0.29",
+    "gatsby-plugin-react-helmet": "^3.0.12",
+    "gatsby-plugin-sass": "^2.0.11",
+    "gatsby-source-filesystem": "^2.0.29",
+    "node-sass": "^4.12.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-helmet": "^5.2.0"
+  },
+  "keywords": [
+    "paragon",
+    "pattern library",
+    "design",
+    "react",
+    "component library"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "start": "npm run develop",
+    "lint": "eslint --ext .js --ext .jsx .",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Its not super important at the moment, but adding a `gatsby clean` to the scripts will come in handy. It can help fix a few errors when content isn't updating properly.

Reply:
Added.